nvm.sh

```diff
@@ -281,27 +281,30 @@ nvm_find_up() {
   nvm_echo "${path}"
 }
 
-
 nvm_find_nvmrc() {
   local dir
   dir="$(nvm_find_up '.nvmrc')"
   if [ -e "${dir}/.nvmrc" ]; then
     nvm_echo "${dir}/.nvmrc"
+  else
+    dir="$(nvm_find_up '.node-version')"
```

Comment:
The semantic here then would be "if no nvmrc is found *anywhere*, look for a node-version" - i'd expect `nvm_find_up` to need to take multiple filenames.

In other words:
```
.nvmrc
   .node-version
      $PWD
```
in $PWD, I'd expect the node-version file to take precedence.

Whereas in:
```
.nvmrc
   .node-version
   .nvmrc
      $PWD
```

I'd expect the inner .nvmrc to take precedence.

Reply:
Done